### ISSUE TITLE
Version 0.8.26 doesn't start

### ISSUE THREAD

**Issue by ypcatify** [voltrb/volt#120]
bundle exec volt server
Volt 0.8.26
Using rack adapter
/Users/ypcat/.rvm/gems/ruby-2.1.3@volt/gems/volt-0.8.26/lib/volt/server/rack/asset_files.rb:38:in `component': Unable to find component 'rspec-builder.rb', make sure the gem is included in your Gemfile (RuntimeError)

**Comment by ryanstout** [comment]
What os and version are you on?

**Comment by rhgraysonii** [comment]
Confirmed to run fine for me on ubuntu 14.04 with ruby 2.2.0 and volt 0.8.27beta3, so upgrading to the latest version of the gem may fix things @ypcatify , so if youre on Linux that could fix it. If OSX it still may help as well.

**Comment by ypcatify** [comment]
Sorry for late response, I'm using OSX Yosemite with Ruby ruby-2.1.3.

**Comment by x3qt** [comment]
Tested with ruby 2.2.0 under OS X 10.10.2. Everything works fine.

**Comment by ypcatify** [comment]
Upgrading to Ruby 2.2.0 helped. Thank you.

**Comment by d4rky-pl** [comment]
Doesn't work for me, fresh ruby 2.2.1 with newly created gemset, OS X 10.10.2

**Comment by d4rky-pl** [comment]
Problem is with `component_paths.rb` method `app_folders`. It does this:

```ruby
app_folders += Gem.loaded_specs.values.map(&:full_gem_path).reject { |g| g !~ /volt/ }.map { |f| f + '/app' }
```

This would probably be a great idea if I didn't use `volt` as gemset name ;) This causes, like, *everything* to add itself to app_folders and because `opal-rspec-0.3.0.beta3/app` actually exists, it adds file inside to load asset loading path and crashes.

The easiest solution I can think of is:

```ruby
app_folders += Gem.loaded_specs.values.reject { |g| g.name !~ /volt/ }.map(&:full_gem_path).map { |f| f + '/app' }

# or a bit nicer
app_folders += Gem.loaded_specs.values.map { |g| File.join(g.full_gem_path, 'app') if g.name =~ /volt/ }.compact

# or even nicer (because of single pass) but slightly less readable
app_folders += Gem.loaded_specs.values.reduce([]) { |paths, gem| paths << File.join(gem.full_gem_path, 'app') if gem.name =~ /volt/; paths }
```

If you want me to send a pull request (maybe with additional test or two?), I'll be happy to provide one after wroclove.rb :)

**Comment by ryanstout** [comment]
@d4rky-pl awesome you figured it out.  I hadn't been able to duplicate this on my end and I don't think anyone put together that a volt gemset was the problem.  A PR would be great.  I'll be at wroclove, so see you there :-)

**Comment by jhirbour** [comment]
I ran into the same issue (OS x yosemite using rvm and ruby-2.1.5) @d4rky-pl fixe looks like a cleaner way than I did it, but the principle was the same.

**Comment by nickmlr** [comment]
@d4rky-pl  - Good call.

**Comment by ryanstout** [comment]
I believe this is fixed in 0.9, please reopen if its not.
